tools/gen-device-svd/gen-device-svd.go

```diff
@@ -124,6 +124,10 @@ type Bitfield struct {
 	value       uint32
 }
 
+func formatDescription(text string) string {
```

Comment:
Maybe this would be better named something like `formatMultiLineText`? Not sure. Every comment field in the SVD is called "description", so using `formatDescription` for multi-line text seems confusing to me.

Or maybe `formatText(text string, multiline bool)`.